manual/auto_correct.md

```diff
@@ -0,0 +1,28 @@
+## Auto-correct
+
+In auto-correct mode, RuboCop will try to automatically fix offenses:
+
+```sh
+$ rubocop -a
+```
+
+For some offenses, it is not possible to implement automatic correction. 
+
+Some automatic corrections that _are_ possible have not been implemented yet.
+
+### Safe auto-correct
+
+```sh
+$ rubocop --safe-auto-correct
+```
+
+In 2018, RuboCop began to annotate cops as `Safe` or not safe.
```

Comment:
I think that writing `RuboCop 0.60` rather than `2018` will be a permanent document.